### ISSUE TITLE
MOZ 2011-15

### ISSUE THREAD

**Issue by eclee25** [HopkinsIDD/cholera-mapping-pipeline#421]

**Comment by QLLZ** [comment]
Data pull:
HASH:  `6772ea9da4f43ddc3da11c494a573af2ca3dd3c8`
[config](https://github.com/HopkinsIDD/cholera-configs/tree/master/no_covariate_production_2023/2011_2015_country)

**Comment by QLLZ** [comment]
Model run:
HASH: `6772ea9da4f43ddc3da11c494a573af2ca3dd3c8`

**Comment by QLLZ** [comment]
[Country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/tree/2023_nocovar_production_runs/country_data_reports)

**Comment by eclee25** [comment]
sd_w and rhat have some convergence issues, lots of Rhat issues for genquant. Rate maps look oddly uniform within a single year, but observation fits look okay.

Opinion: Convergence issues and rate maps need further investigation.

**Comment by javierps** [comment]
The issue here I think is that there is very little data at the subnational level, and the country is quite big. Note the very small range of w values. 

I suspect that all sub-national data is low values compatible with a uniform incidence rate in those years. Would suggest making admin-level plots for the validation figures.

**Comment by javierps** [comment]
Suggestion: no-W run given the low amount of sub-national data with respect to the size of the country.

**Comment by eclee25** [comment]
Rerun on dev (with spatial_effect: yes) now that the DAGAR has been fixed.

**Comment by QLLZ** [comment]
model run

HASH ` f56d5a942b106ad7750b9be31c0456cefa5b6dc7`

**Comment by QLLZ** [comment]
[Updated configs](https://github.com/HopkinsIDD/cholera-configs/tree/master/no_covariate_production_2023/round3/moz_2011_2015)

**Comment by QLLZ** [comment]
Data pull and model run:
HASH: ` fb20b87bab8e4c41fe520de2dc76de0050ece361`

**Comment by QLLZ** [comment]
[Country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/2023_nocovar_production_runs/country_data_reports/MOZ_2011-01-01_2015-12-31_prd-2023-rd3_4a5de11f1ba28ad2c98f763e563469d5.country-data-report.html)

**Comment by eclee25** [comment]
Continued convergence issues on the sd_w and w traceplots, which may be related to the mixture prior on sd_w. We will try to relaunch this on dev_nomixture_sd_w_prior branch with the config setting `do_sd_w_mixture: no`

**Comment by eclee25** [comment]
When do_sd_w_mixture: no the sd_w trace plots looked better although not perfect. This setting can be an exception to a standard run -- designed for countries with only limited subnational data.

**Comment by eclee25** [comment]
Moving this back to standard run because we are changing the mixture prior on sd_w to have a stronger weight on lower values

**Comment by QLLZ** [comment]
[Config](https://github.com/HopkinsIDD/cholera-configs/blob/master/July_2023_nocovar_production_runs/standard_runs/config_prd-jul2023_MOZ_2011_2015.yml)

Data pull & Model run:

HASH: 211913e0eb7d570ed62c48bc1ae9ecd6671a6cbc

**Comment by QLLZ** [comment]
[country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/July_2023_nocovar_production_runs/country_data_reports/MOZ_2011-01-01_2015-12-31_prd-jul2023_3f6348c8f9e866a6f2aa4c5606d00a00.country-data-report.html)

**Comment by QLLZ** [comment]
Bad convergency (few subnational data). Modeled cases look good. 

Suggestion: maybe try other settings?

**Comment by eclee25** [comment]
Convergence issues with w and sd_w even after mixture prior weight was shifted lower. 
Opinion: Rerun with `do_sd_w_mixture: no`

**Comment by QLLZ** [comment]
[Updated config](https://github.com/HopkinsIDD/cholera-configs/blob/master/July_2023_nocovar_production_runs/exception_runs/config_prd-jul2023_MOZ_2011_2015.yml)

**Comment by QLLZ** [comment]
[Updated country data report](https://github.com/HopkinsIDD/cholera-mapping-reports/blob/July_2023_nocovar_production_runs/country_data_reports/MOZ_2011-01-01_2015-12-31_prd-jul2023_6677fed184c673182e2eba8fcd3cbe1c.country-data-report.html)

**Comment by eclee25** [comment]
MOZ traceplots are much improved even if the non-mixture sd_w prior hasn’t completely resolved everything.

**Comment by javierps** [comment]
Agreed, the Rhats are higher not super large, and I don't think we will be able to do much better here unless we put a stronger prior on std_dev_w.

**Comment by eclee25** [comment]
ID-57 run is approved. For future runs, we need to have `do_sd_w_mixture: no`
Approve ID-57

**Comment by QLLZ** [comment]
Data pull, model run:

HASH: `05f598b95b5d42ac0049963f8a0b7e04d6f1dbd7`

**Comment by javierps** [comment]
Rho convergence not perfect, std_w OK, ws seem to also have some differences. Rhats all OK.

Suggestion: Accept.

**Comment by eclee25** [comment]
Approve
